<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pacca - Programa de Análise do Comportamento Animal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/logo48x48.png</normaloff>:/icons/logo48x48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNova">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Nova</string>
    </property>
    <widget class="QMenu" name="menuSess_o">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Sessão</string>
     </property>
     <addaction name="actionText"/>
     <addaction name="actionAudio"/>
     <addaction name="actionVideo"/>
    </widget>
    <addaction name="menuSess_o"/>
    <addaction name="separator"/>
    <addaction name="actionAnalise"/>
   </widget>
   <widget class="QMenu" name="menuControle">
    <property name="title">
     <string>Controle</string>
    </property>
    <widget class="QMenu" name="menuJanelas">
     <property name="title">
      <string>Janelas</string>
     </property>
     <addaction name="actionCascata"/>
     <addaction name="actionLado_a_Lado"/>
    </widget>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionDicion_rio"/>
    <addaction name="actionSpecies"/>
    <addaction name="actionSess_es"/>
    <addaction name="actionUsuario"/>
    <addaction name="separator"/>
    <addaction name="actionReiniciar_Banco"/>
    <addaction name="separator"/>
    <addaction name="menuJanelas"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionIdioma"/>
    <addaction name="separator"/>
    <addaction name="aboutPacca"/>
    <addaction name="aboutQt"/>
   </widget>
   <addaction name="menuNova"/>
   <addaction name="menuControle"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUsuario">
   <property name="text">
    <string>Usuário</string>
   </property>
   <property name="shortcut">
    <string>Meta+U</string>
   </property>
  </action>
  <action name="actionReiniciar_Banco">
   <property name="text">
    <string>Reiniciar Banco</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
   <property name="shortcut">
    <string>Meta+V</string>
   </property>
  </action>
  <action name="actionText">
   <property name="text">
    <string>Arquivo</string>
   </property>
   <property name="shortcut">
    <string>Meta+F</string>
   </property>
  </action>
  <action name="actionAnalise">
   <property name="text">
    <string>Análise</string>
   </property>
   <property name="shortcut">
    <string>Meta+G</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
   <property name="shortcut">
    <string>Meta+L</string>
   </property>
  </action>
  <action name="actionDicion_rio">
   <property name="text">
    <string>Dicionários</string>
   </property>
   <property name="shortcut">
    <string>Meta+D</string>
   </property>
  </action>
  <action name="actionSess_es">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sessões</string>
   </property>
   <property name="shortcut">
    <string>Meta+S</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="text">
    <string>Audio</string>
   </property>
   <property name="shortcut">
    <string>Meta+A</string>
   </property>
  </action>
  <action name="actionSpecies">
   <property name="text">
    <string>Espécies</string>
   </property>
   <property name="shortcut">
    <string>Meta+E</string>
   </property>
  </action>
  <action name="aboutPacca">
   <property name="text">
    <string>Sobre o Pacca</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>Sobre o Qt</string>
   </property>
  </action>
  <action name="setENG_US">
   <property name="text">
    <string>Inglês (US)</string>
   </property>
  </action>
  <action name="setPT_BR">
   <property name="text">
    <string>Português (BR)</string>
   </property>
  </action>
  <action name="actionIdioma">
   <property name="text">
    <string>Idioma</string>
   </property>
  </action>
  <action name="actionCascata">
   <property name="text">
    <string>Cascata</string>
   </property>
  </action>
  <action name="actionLado_a_Lado">
   <property name="text">
    <string>Lado a Lado</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
